src/main.js

```diff
@@ -23,7 +23,9 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 
 /* import specific icons */
 library.add(faUserSecret, faInstagram, faFacebookF, faTwitter, faYoutube)
+
 /* add font awesome icon component */
+// eslint-disable-next-line vue/component-definition-name-casing
```

Comment:
@ahn-nath I'd like you to look at this. This allows us to ignore the lint errors for the next line. For exceptional cases, this is what we use.

Pro tip: If you're not able to commit/push your changes due to link errors failing in the push/commit hook, you can always use `git commit --no-verify` or `git push -n` to temporarily skip the hooks.